File: src/main/java/no/nav/pensjon/innsyn/domain/desc/ForstegangstjenesteDescriptor.java
package no.nav.pensjon.innsyn.domain.desc;

import no.nav.pensjon.innsyn.entity.EntityDescriptor;

public class ForstegangstjenesteDescriptor implements EntityDescriptor {

    private static final String ENTITY_NAME = "Førstegangstjeneste";

    private static final String[] PROPERTY_NAMES = {
            "Tjenestestart",
            "Dimittert",
            "Rapporttype",
            "Status",
            "Kilde"
    };

    public String getEntityName() {
        return ENTITY_NAME;
    }

    public String[] getPropertyNames() {
        return PROPERTY_NAMES;
    }
}

File: src/main/kotlin/no/nav/pensjon/innsyn/domain/desc/ForstegangstjenesteDescriptor.kt
package no.nav.pensjon.innsyn.domain.desc

import no.nav.pensjon.innsyn.domain.Forstegangstjeneste
import no.nav.pensjon.innsyn.entity.EntityDescriptor

object ForstegangstjenesteDescriptor : EntityDescriptor<Forstegangstjeneste>("Førstegangstjeneste",
        arrayOf(
                "Tjenestestart",
                "Dimittert",
                "Rapporttype",
                "Status",
                "Kilde"
        )
)